polaris/evaluate/_metric.py

```diff
@@ -65,6 +88,7 @@ class Metric(Enum):
     pearsonr = MetricInfo(fn=pearsonr, direction="max")
     spearmanr = MetricInfo(fn=spearman, direction="max")
     explained_var = MetricInfo(fn=explained_variance_score, direction="max")
+    aafe = MetricInfo(fn=absolute_average_fold_error, direction="max")
```

Comment:
For clarity, I would use:
```suggestion
    absolute_average_fold_error = MetricInfo(fn=absolute_average_fold_error, direction="max")
```